### DISCUSSION TITLE
lolMiner 1.26

### DISCUSSION THREAD

**Discussion by TorGj** [Lolliedieb/lolMiner-releases#437] [General]
1.21 and 1.25 worked (works) like a charm.

1.26 ... I'm not allowed to start it with the bat-file...
I've tried on 3 machines: 1060, 1070 and 2070 ... all fail.

**Comment by cryp-tog** [comment]
Antivirus kicked in, and "helped" me by blocking it due to recognition as, "caypnamer", which wasn't an issue going from 1.21 to 1.25.  You'll need to unblock/allow the exe and bat to run.  Have you done this?

**Comment by elShimmer** [comment]
Hey, I made a switch to Trex Miner after Lol Miner 1.21 stopped working for me. The 1.26 does work for me, but with my 3090 I'm getting 121-123 average vs 113 on lol. Just throwing the difference in hash rate out there.

**Comment by MiroslauF** [comment]
1.21 was good. Running weeks without issue. 1.25 crashes after 12-15 hours... 1.26 works fine. Little bit higher hashrate. What is important that GPU core temperature is lower -2°C. I am using Nvidia cards. (2x1070 - 66°C, 1080Ti - 54°C, 2070 - 56°C), OS - Ubuntu.
